engine/src/w32printer.cpp

```diff
@@ -1342,7 +1342,9 @@ MCPrinterResult MCWindowsPrinterDevice::Bookmark(const char *title, double x, do
 void MCWindowsPrinter::DoInitialize(void)
 {
 	m_valid = false;
-	m_name = MCValueRetain(kMCEmptyString);
+	// SN-2015-03-18: [[ Win shutdown crash ]] Avoid memory leak at printer
+	//  initialisation
+	MCValueAssign(m_name, kMCEmptyString);
```

Comment:
I can see this would cause a memory-leak but *only* if m_name has not been released before DoInitialize() is called...